css/styles.css

```diff
@@ -497,10 +497,6 @@ section#developers .section-header {
 
```

Comment:
We need it here as well as the styles.css is not copied by the doc build (shall we copy it as well?)